mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -916,6 +916,42 @@ func TestAccBigQueryExternalDataTable_CSV_WithSchema_UpdateToConnectionID(t *tes
 	})
 }
 
+func TestAccBigQueryExternalDataTable_CSV_WithSchema_UpdatAllowQuotedNewlines(t *testing.T) {
```

Comment:
Is this intended to have "Updat" instead of "Update"?

Reply:
Of course, not :)